blazor/rich-text-editor/import-export.md

```diff
@@ -44,7 +44,7 @@ The Rich Text Editor allows you to load an external HTML file in the editor's co
 {% endhighlight %}
 {% endtabs %}
 
-![Blazor RichTextEditor import to HTML file](./images/blazor-richtexteditor-import-html.png)
+{% previewsample "" %}
```

Comment:
why this link was empty?

Reply:
Hi @rajendranr-5483 ,

I corrected this. This file was wrongly committed. Because, reload and code issues. So, i marked this. Now, i removed this file changes.

Regards,
@Backiaraj